server/data/wikidata/run_query.js

```diff
@@ -54,9 +62,18 @@ const parametersTests = {
 
 const runQuery = (params, key) => {
   const { query: queryName } = params
-  const { query: queryBuilder } = queries[queryName]
-  const sparql = queryBuilder(params)
+  const sparql = queries[queryName].query(params)
 
   return makeSparqlRequest(sparql)
   .catch(_.ErrorRethrow(key))
 }
+
+radio.on('invalidate:wikidata:entities:relations', async ({ property, valueUri }) => {
+  const queriesToInvalidate = queriesPerProperty[property] || []
+  queriesToInvalidate.push(...queriesPerProperty['*'])
```

Comment:
this `'*'` looks cryptic, could we leave a comment explaining what does this do ?

Reply:
Indeed. This is used for queries that should be invalidated for any value of 'property': would writing that in a comment be enough or is there also a better way to express that than `*`?